src/layout/layout.ts

```diff
@@ -68,7 +68,7 @@ class Layout implements IComponent {
   // them with real data asynchronously (if it takes too long to do at once) by yielding a thread
   // and returning to it later through a set timeout
   private discoverDom() {
-    let discoverTime = getPageContextBasedTimestamp();
+    let discoverTime = Math.round(getPageContextBasedTimestamp());
```

Comment:
I'm not suggesting you add the round() to this function but create a separate function that also does the rounding and returns that rounded time rather than putting that multiple places in your code.